com.regnosys.rosetta.tests/META-INF/MANIFEST.MF

```diff
@@ -5,8 +5,8 @@ Bundle-Name: com.regnosys.rosetta.tests
 Bundle-SymbolicName: com.regnosys.rosetta.tests;singleton:=true
 Bundle-Version: 0.0.0.master
 Bundle-Vendor: REGnosys
-Require-Bundle: org.junit.jupiter.api;bundle-version="[5.1.0,6.0.0)",
- org.eclipse.xtext.xbase.lib;bundle-version="2.25.0",
+Require-Bundle: org.junit.jupiter.api,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
```

Comment:
Shouldn't this one be on 2.27.0? (same as Xtext)

Reply:
Updated